ci/test-checks.sh

```diff
@@ -13,6 +13,15 @@ export RUSTFLAGS="-D warnings"
 # Look for failed mergify.io backports
 _ git show HEAD --check --oneline
 
+if _ scripts/cargo-for-all-lock-files.sh check --locked; then
```

Comment:
This is what I really want to do before any our depending crates are compromised. :)